lib/components/Button/style.scss

```diff
@@ -50,7 +50,8 @@ $widthSmall: 36px;
 	text-decoration: none;
 	cursor: pointer;
 	user-select: none;
-	transform: translate(0, 0);
+	transform: translate(0, 0) scale(1);
+	will-change: transform;
```

Comment:
Not compatible for all browsers but still good enough:
https://caniuse.com/#feat=will-change

Reply:
It's more a case of "if this property exists" it'll be good if you honor it, but if it doesn't exist, it wont break things. So all good :100: